### PR TITLE
Add components do busbar show

### PR DESCRIPTION
**Pull request by waldman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Unreleased
+### Add
+- Add components do busbar show
```
[the remark this answers is not in the document]
Comment (waldman) [reply CHANGELOG.md]: LOL... fixing! =P

File: lib/busbar_cli/models/environment.rb [hunk 2]
```diff
@@ -36,4 +38,12 @@ def pretty_settings
       "\t#{setting}: #{value}"
     end.join("\n")
   end
+
+  def pretty_components
+    components.map do |component, _|
+      component.map do |attribute, value|
```
Comment (mattgimbal) [inline lib/busbar_cli/models/environment.rb]: should we `.sort` by the component name (like how `pretty_settings` is sorted?) Don't feel terribly strongly about this, but may be nice to have "`click-web`, `clock`, `web`, `worker`, `viewability_worker`" in that order..what you do think?

**Comment by ilyakam** [comment]
What problem does this PR solve? Please link it to an existing issue if applicable.
What does the new output look like? I'm seeing a lot of user-facing changes (e.g., `\t--`) and would like to see a screenshot / STDOUT of the output.

**Comment by waldman** [comment]
@ilyakam we didn't have access to the data about the components of an specific application. 

With the changes that we did for the Hackathon no the busbar-server this data became available and now can be fetch on the CLI.

Here is an example output:
```
Components:
        type: web
        node_id: 1x.standard
        scale: 3
```

And this is added to the `busbar show` command.

File: lib/busbar_cli/models/environment.rb (hunk 2 above)
Comment (waldman) [reply lib/busbar_cli/models/environment.rb]: Good call... Let me give a look on this in a few! =))

**Comment by ilyakam** [comment]
Excellent! Please use this answer to elaborate on the changes in the commit message's body so that we can always point to the reasons behind the change in `git blame`.

File: lib/busbar_cli/models/environment.rb (hunk 2 above)
Comment (waldman) [reply lib/busbar_cli/models/environment.rb]: Done! =))

**Pull request merged**